crates/fuel-core/src/service/adapters/block_importer.rs

```diff
@@ -72,11 +72,7 @@ impl BlockImporterAdapter {
         &self,
         sealed_block: SealedBlock,
     ) -> anyhow::Result<()> {
-        tokio::task::spawn_blocking({
-            let importer = self.block_importer.clone();
-            move || importer.execute_and_commit(sealed_block)
-        })
-        .await??;
+        self.block_importer.execute_and_commit(sealed_block).await?;
```

Comment:
Now the importer uses `tokio_rayon` inside.

Reply:
perfect